bin/main.rs

```diff
@@ -1,15 +1,16 @@
 //! Oasis Eth/WASI runtime entry point.
 extern crate oasis_ethwasi_runtime;
 
+extern crate anyhow;
 extern crate ethcore;
 extern crate ethereum_types;
-extern crate failure;
 extern crate io_context;
 extern crate oasis_core_keymanager_client;
 extern crate oasis_core_runtime;
 extern crate oasis_ethwasi_runtime_api;
 extern crate oasis_ethwasi_runtime_common;
 extern crate serde_bytes;
+extern crate thiserror;
```

Comment:
this doesn't work (at least for `thiserror:Error` one) since it uses [procedural macros](https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html#procedural-macros)